src/components/profile/team/GQLYourApprovals.js

```diff
@@ -253,51 +223,37 @@ class GQLYourApprovals extends React.Component {
           if (loading) return 'Loading...';
           if (error) return `Error!: ${error}`;
           const approvalData = (!data) ? '' : data.approvals;
-          const aList = (approvalData.length > 0)
-            ? approvalData.map(apprvl => (
-              <ApprovalList
-                key={apprvl.id}
-                user={apprvl.gcIDSubmitter}
-                approvalID={apprvl.id}
-                toggle={(e) => { this.toggle(e); }}
-                activeTab={this.state.activeTab}
-                status={apprvl.status}
-              />
-            )) : 'NO APPROVALS';
-
           const aPane = (approvalData.length > 0)
             ? approvalData.map(apprvl => (
-              <ApprovalPane
-                key={apprvl.id}
-                changeType={apprvl.changeType}
-                approval={
-                  {
-                    id: apprvl.id,
-                    user: {
-                      gcID: apprvl.gcIDSubmitter.gcID,
-                      name: apprvl.gcIDSubmitter.name,
-                    },
+              <div key={apprvl.id}>
+                <ApprovalList
+                  user={apprvl.gcIDSubmitter}
+                  status={apprvl.status}
+                />
+                <ApprovalPane
+                  changeType={apprvl.changeType}
+                  approval={
+                    {
+                      id: apprvl.id,
+                      user: {
+                        gcID: apprvl.gcIDSubmitter.gcID,
+                        name: apprvl.gcIDSubmitter.name,
+                      },
+                    }
                   }
-                }
-                requestedChange={apprvl.requestedChange}
-              />
-            )) : '';
+                  requestedChange={apprvl.requestedChange}
+                />
+              </div>
+            )) : 'NO APPROVALS';
```

Comment:
Since we are modifying the approval UI, we should create a nicer no approvals screen.